docs/user-guide.md

```diff
@@ -112,42 +120,3 @@ Then run the following to connect:
 cqlsh `docker-machine ip default` --cqlversion=3.2.0
 ```
 
-## Zipkin's internal storage structure (in Cassandra)
```

Comment:
Do you mean you'd rather keep this section?
I thought I'd remove it since the schema definition file reads well enough and the most current doc is always the code.
I don't mind making a separate PR to openzipkin/zipkin with that if you think it'd be better to keep it.